e2e/pnpm_lockfiles/lockfile-test.bzl

```diff
@@ -19,6 +19,9 @@ BZL_FILES = {
     # global
     "defs.bzl": "@REPO_NAME//:defs.bzl",
 
+    # resolved.json reference
+    "is-odd_resolved.json": "@REPO_NAME//VERSION:is-odd/resolved.json",
```

Comment:
This just asserts `resolved.json` is generated in the expected spot, previously it was generated in `:VERSION/is-odd/resolved.json`